### ISSUE TITLE
Customizer: add options to change font size and family

### ISSUE THREAD

**Issue by Mayaerin1** [WordPress/gutenberg#8788]
I am frequently frustrated by the font restrictions for different themes. I want to use the page template with Sela because of it's mobile compatibility but change the font types and sizes.

I would like a feature added to the customizer that allows the editing of the font presets (Heading One, Heading Two, etc.) themselves. So instead of having to create custom CSS for every font change, I could set the font family, size, color, weight, transformation for each Header type. The existing interface with the limited text type drop-down choices could still be used but it would be following user presets instead of the theme's presets.

Right now any changes to font have to be added to custom CSS like:

h5 {font-size:100%; font-family:"open sans",sans-serif}

h6 {font-size:100%; font-family:"open sans",sans-serif}

So it would be awesome if that could just be converted into an option in the customizer tabs.

**Comment by pgiblett** [comment]
Gutenberg is supposed to be more of a WYSWYG editor than the previous version. Yet is has not improved the font options. One of the limitations of Gutenberg plugin it that is does not provide options for adding fonts. It has a limited capability to set font size. 

To be fair the prior editor doesn't have fonts either, but I have installed a plugin to provide me with fonts.

You have made great strides, but without fonts you are only halfway there. I may change fonts within a post when demonstrating items. I have found this to be very useful with posts I create.
